hi-ml-histopathology/src/histopathology/datamodules/base_module.py

```diff
@@ -85,6 +85,15 @@ def get_splits(self) -> Tuple[_SlidesOrTilesDataset, _SlidesOrTilesDataset, _Sli
         """Create the training, validation, and test datasets"""
         raise NotImplementedError
 
+    def train_dataloader(self) -> DataLoader:
+        return self._get_dataloader(self.train_dataset, shuffle=True, stage=ModelKey.TRAIN, **self.dataloader_kwargs)
+
+    def val_dataloader(self) -> DataLoader:
+        return self._get_dataloader(self.val_dataset, shuffle=True, stage=ModelKey.VAL, **self.dataloader_kwargs)
+
+    def test_dataloader(self) -> DataLoader:
+        return self._get_dataloader(self.test_dataset, shuffle=True, stage=ModelKey.TEST, **self.dataloader_kwargs)
```

Comment:
Now that I look at these - I think it's important we shuffle the data before splitting in train/test/validation but for reproducibility we should not randomly shuffling val and test - thoughts?

Reply:
Yes definitely! I wanted to point it out as well!

Reply:
we should also maybe consider setting random_offset to False for val/test. what do you think?

Reply:
shall we change? @harshita-s would you agree?

Reply:
Sure we can change it, I saw a warning on AML as well

Reply:
"we should also maybe consider setting random_offset to False for val/test. what do you think?" -> yes, good point